test/integration/admin_chips_test.rb

```diff
@@ -47,4 +47,41 @@ def create_admin
     assert_equal admin_chips_path, current_path
     assert page.has_content? 'EditedName'
   end
+
+  test 'admin can add chip' do
+    create_admin
+    category_1 = Oil.create(name: "Lard")
+    ApplicationController.any_instance.stubs(:current_user).returns(@admin)
+
+    visit admin_chips_path
+    click_link 'Add New Chip'
+    fill_in 'Name', with: 'NewChip'
+    fill_in 'Price', with: 1.99
+    fill_in 'Description', with: 'Coolest of the chips'
+    click_button 'Create Chip'
+
+    assert admin_chips_path, current_path
+
+    within('.chips') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.